src/de.rs

```diff
@@ -258,7 +258,7 @@ impl<'a, 'de> de::Deserializer<'de> for &'a mut Depythonizer<'de> {
                 .get_item(0)?
                 .downcast()
                 .map_err(|_| PythonizeError::dict_key_not_string())?;
-            let value = d.get_item(variant).unwrap();
+            let value = d.get_item(variant).unwrap().unwrap();
```

Comment:
I am not 100% sure this is the correct way to handle such case.

Reply:
I think this can work, and if so, is better:

```suggestion
            let value = d.get_item(variant)?.unwrap();
```

Reply:
It seems to work :+1: Thanks for the suggestion.